runtime/gc_modron_startup/mminit.cpp

```diff
@@ -274,7 +274,10 @@ gcCleanupHeapStructures(J9JavaVM * vm)
 		gam->flushAllocationContextsForShutdown(&env);
 	}
 
-	if (!IS_RESTORE_RUN(vm)) {
+#if defined(J9VM_OPT_SNAPSHOTS)
+	if (!IS_RESTORE_RUN(vm))
+#endif /* defined(J9VM_OPT_SNAPSHOTS) */
```

Comment:
Cases as this where code block size / inlining / dead-code determination is not impacted can be skipped.

Reply:
Removed the change to this and other similar cases.